.github/ISSUE_TEMPLATE/feature-change.yml

```diff
@@ -14,25 +14,32 @@ body:
   # User story sentence
   - type: textarea
     attributes:
-      label: User Story Sentence
+      label: What change would you like to see?
+      description: Describe the solution you'd like. If relevant, include ways in which you've tried to solve the issue with the current version.
       placeholder: "As a user, I want to be able to ____ so that I can ____"
     validations:
       required: true
   # Requirements
   - type: textarea
     attributes:
       label: Requirements
-      description: List the outcomes of the feature being implemented without design or implementation details.
+      description: |
+        Webrecorder team only! :)
+        
+        List the outcomes of the feature being implemented without design or implementation details.
       placeholder: |
         1. Item metadata should show links to the collections that the item belongs to.
         2. Items can be added or removed from collections when editing an item.
     validations:
-      required: true
+      required: false
   # Todo
   - type: textarea
     attributes:
       label: Todo
-      description: Any other linked issues / tasks to complete to implement this feature (leave blank if unknown).
+      description: |
+        Webrecorder team only! :)
```

Comment:
What if we just say: 
(Intended primarily for Webrecorder team, leave blank if unknown)